src/frontend-samples/validation-sample/ValidationApi.ts

```diff
@@ -0,0 +1,191 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
This whole file is unnecessary since we're using the json static data correct?
We're just keeping it around until the validation API is working again? I'm assuming this file is kept for educational purposes to demonstrate a 'real-world' use-case.

Reply:
Yes the ValidationClient.ts file isn't being used right now. It's just to show how you would actually make an api call if there was validation rules set up using the imodel. Once the Validation API changes are in prod, which I've heard should be this week, I'll update this sample to actually make use of this file.

Reply:
Sounds good